src/main/java/lotto/model/WinningLotto.java

```diff
@@ -0,0 +1,34 @@
+package lotto.model;
+
+import java.util.Map;
+import java.util.stream.Collectors;
+import lotto.model.number.LottoNumber;
+import lotto.model.number.LottoNumbers;
+
+public class WinningLotto {
+
+    private static final String DUPLICATED_NUMBER_ERROR_MESSAGE = "[ERROR] 중복된 번호가 존재합니다.";
+
+    private final LottoNumbers winningNumbers;
+    private final LottoNumber bonusNumber;
+
+    public WinningLotto(final LottoNumbers winningNumbers, final LottoNumber bonusNumber) throws RuntimeException {
+        this.winningNumbers = winningNumbers;
+        checkDuplicateNumber(bonusNumber);
+        this.bonusNumber = bonusNumber;
+    }
+
+    public void checkDuplicateNumber(LottoNumber number) throws RuntimeException {
+        if ((winningNumbers.getLottoNumbers().stream()
+                .mapToInt(LottoNumber::getLottoNumber)
+                .boxed()
+                .collect(Collectors.toList()))
+                .contains(number.getLottoNumber())) {
```

Comment:
조건식에 stream 체인이 들어가면 어떤 결과가 나오는지 알아야 조건식을 이해할 수 있겠네요.

아래처럼 나타내면 어떨까요?

```suggestion
        boolean isBonusNumberDuplicate =
                winningNumbers.getLottoNumbers().stream()
                        .anyMatch(i -> i.getLottoNumber() == number.getLottoNumber());
        if (isBonusNumberDuplicate) {
```

Reply:
감사합니다 :>

Reply:
getter를 사용하지 않도록 코드를 수정하였는데요, 위에서 피드백해주신 getter를 사용할 때, 새로운 컬렉션으로 반환하라고 해주셨는데, 이 를 활용할때는 view 외의 코드에 사용해도 될까요??

Reply:
어떤 의미인지 잘 이해가 안되는데, 조금만 자세히 설명해주실 수 있을까요?

Reply:
getter의 사용을 지양하라는 말에서 getter로 객체를 조작할 수도 있으므로 지양하라는 것으로 생각되는데, unmodifable Collection과 같이 새로운 컬렉션으로 반환되는 getter는 view를 제외한 코드에서 사용을 해도 되는 것일까요??

Reply:
컬렉션의 항목도 immutable하다면 괜찮지 않을까요?